install/powershellscript/microsoft-go-install.ps1

```diff
@@ -0,0 +1,898 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+    Installs Microsoft Go
+.DESCRIPTION
+    Installs the Microsoft Go toolset.
+
+    Note that the intended use of this script is for Continuous Integration (CI) scenarios, where:
+    - The toolset needs to be installed without user interaction and without admin rights.
+    - The toolset installation doesn't need to persist across multiple CI runs.
+    Visit https://github.com/microsoft/go for a list of other ways to install Microsoft Go.
+
+.PARAMETER Version
+    Default: Latest
+    Download the specified version. Supports some aliases. Possible values:
+    - Latest - the most recent major version.
+    - Previous - the second most recent major version.
+    - 2-part version in format go1.A - represents a specific major version.
```

Comment:
We should document whether prerelease versions are supported or not, i.e. `1.23rc1`.